doc/architecture-design.md

```diff
@@ -45,15 +45,17 @@
 
 ## APISIX Config
 
-For example, set the default listening port of APISIX to 8000, and keep other configurations as default. The configuration in `conf/config.yaml` should be like this:
+There are two methods to configure APISIX: directly change `conf/config.yaml`, or add file path argument when start APISIX like `apisix start -c /opts/apisix/config.yaml`
```

Comment:
Better to mention `--config` option too.

Reply:
fixed thx